CleverTap-iOS-SDK.podspec

```diff
@@ -10,14 +10,15 @@ s.requires_arc              = true
 s.module_name               = 'CleverTapSDK'
 s.resources                 = 'CleverTapSDK/*.{cer}'
 s.resource_bundles          = {'CleverTapSDK' => ['CleverTapSDK/*.{xcprivacy}']}
+s.swift_version             = '5.0'
 s.ios.dependency             'SDWebImage', '~> 5.11'
 s.ios.resource_bundle       = {'CleverTapSDK' => ['CleverTapSDK/**/*.{png,xib,html}', 'CleverTapSDK/**/*.xcdatamodeld']}
 s.ios.deployment_target     = '9.0'
-s.ios.source_files          = 'CleverTapSDK/**/*.{h,m}'
+s.ios.source_files          = 'CleverTapSDK/**/*.{h,m,swift}'
```

Comment:
Do we need to put Swift files into a separate group/folder, or does it not matter? I don't recall if there were any limitations.

Reply:
the other 2 branches which use swift classes seem to work fine with this config. Seems ok.